sys/include/fido2/ctap/ctap_utils.h

```diff
@@ -47,26 +47,29 @@ int fido2_ctap_utils_init_gpio_pin(gpio_t pin, gpio_mode_t mode, gpio_flank_t fl
  *
  * Successful if user clicks button in less than @ref CTAP_UP_TIMEOUT
  *
- * @return @ref ctap_status_codes_t
+ * @return @ref ctap_status_code_t
  */
 int fido2_ctap_utils_user_presence_test(void);
 
 /**
- * @brief Compare fido2 credentials based on creation time
+ * @brief Compare fido2 credentials based on id to find more recent one
+ *
+ * The more recent credential has a higher id. Therefore we sort in
+ * descending order.
  *
  * @param[in] k1    first resident key
  * @param[in] k2    second resident key
  *
- * @return <0 if k2 has a bigger sign_count
- * @return 0 if equal k1 and k2 have equal sign_count
- * @return >0 if k1 has a bigger sign_count
+ * @return <0 if k1 > k2
+ * @return >0 if k1 < k2
```

Comment:
It's in the text that we compare by ID, but still seeing pointer arguments as they are in a comparison looks confusing; maybe:
```suggestion
 * @return <0 if ID of k1 > ID of k2
 * @return >0 if ID of k1 < ID of k2
```